### PR TITLE
208/mock orders

### PR DESCRIPTION
**Pull request by alfetopito**
Part of #208 
Depends on #286 and #287 

- Adding mock orders for local testing.
- Added utils time functions to calculate batchIds based on Date objects OR timestamps
- Reordered ExchangeApiMock constructor parameters for convenience

NOT loaded into the Orders page yet

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr288--dexreact.review.gnosisdev.com

File: src/api/exchange/ExchangeApiMock.ts [hunk 1]
```diff
@@ -23,8 +23,8 @@ export class ExchangeApiMock extends DepositApiMock implements ExchangeApi {
     balanceStates: BalancesByUserAndToken,
     erc20Api: Erc20Api,
     registeredTokens: string[] = [],
-    maxTokens = 10000,
     ordersByUser: OrdersByUser = {},
+    maxTokens = 10000,
```
Comment (Velenir) [inline src/api/exchange/ExchangeApiMock.ts]: Is it time to switch to object as params? :smile:

Comment (alfetopito) [reply src/api/exchange/ExchangeApiMock.ts]: Well, can't say that I thought about that. But I like the suggestion. Will do!

File: src/utils/time.ts [hunk 2]
```diff
@@ -1,10 +1,83 @@
...
+ * Example: dateAdd(new Date(), 30, 'minute')  //returns 30 minutes from now.
+ * https://stackoverflow.com/a/1214753/18511
+ *
+ * Adapted from https://stackoverflow.com/a/1214753/1272513
+ *
+ * @param date  Date to start with
+ * @param time  How much of given time unit to add to date.
+ * @param unit  One of: year, month, week, day, hour, minute, second
+ */
+export function addTimeToDate(
```
Comment (anxolin) [inline src/utils/time.ts]: Yes, this ideally uses a library. Here it looks there's some alternatives to momentjs if we feel moment might add to much weight.
https://github.com/you-dont-need/You-Dont-Need-Momentjs

Comment (anxolin) [reply src/utils/time.ts]: For this function I would consider OK not to use a lib, but I think we'll need to format dates easily, like for showing relative dates "fromNow" functionality

Comment (alfetopito) [reply src/utils/time.ts]: Will indeed need that for the next step. Will look into it.

**Pull request merged**